tutor/src/models/reference-book/ux.js

```diff
@@ -83,6 +83,10 @@ export default class BookUX {
       // in that case the id that's present will be the ecosystem
       const course = Courses.get(this.courseId) ||
         Courses.forEcosystemId(this.courseId);
+      if (course.id != this.courseId) {
+        this.courseId = course.id;
+        this.router.history.push(`/book/${this.courseId}`);
```

Comment:
Wouldn't we need the chapter-section here if it's present? And maybe remove the last entry so it does not get duplicated (but I'm not sure about this one).

Reply:
yeah, we will need chapter-section.  good catch, I'll add that